modules/indexExchangeBidAdapter.js

```diff
@@ -419,6 +426,9 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
     _IndexRequestData.targetAggregate = {'open': {}, 'private': {}};
 
     if (!utils.hasValidBidRequest(bidArr[0].params, requiredParams, ADAPTER_NAME)) {
+      for (var i = 0; i < bidArr.length; i++) {
+        passOnBid(bidArr[i].placementCode);
+      }
```

Comment:
@arvending I've tested the fix, and it works (else this was raising the timeout). But @snapwich is right as the `bidRequested` event confirms it.

Nevertheless, I'll patch my 0.26.1 release with your fix, so that I can make it work (I've warned your team a while ago about it, so I'm okay with this temporary solution).

Thanks for your patch anyway.